modules/contrib_rules_jvm/0.21.2/MODULE.bazel

```diff
@@ -0,0 +1,249 @@
+    "com.google.auto.value:auto-value:1.9",
+    "com.google.code.findbugs:jsr305:3.0.2",
+    "com.google.code.gson:gson:2.9.0",
+    "com.google.errorprone:error_prone_annotations:2.9.0",
+    "com.google.guava:failureaccess:1.0.1",
+    "com.google.guava:guava:31.0.1-android",
+    "com.google.j2objc:j2objc-annotations:1.3",
+    "com.google.re2j:re2j:1.6",
+    "com.google.truth:truth:1.0.1",
+    "com.squareup.okhttp:okhttp:2.7.5",
+    "com.squareup.okio:okio:1.17.5",
+    "io.netty:netty-buffer:4.1.87.Final",
+    "io.netty:netty-codec-http2:4.1.87.Final",
+    "io.netty:netty-codec-http:4.1.87.Final",
+    "io.netty:netty-codec-socks:4.1.87.Final",
+    "io.netty:netty-codec:4.1.87.Final",
+    "io.netty:netty-common:4.1.87.Final",
+    "io.netty:netty-handler-proxy:4.1.87.Final",
+    "io.netty:netty-handler:4.1.87.Final",
+    "io.netty:netty-resolver:4.1.87.Final",
+    "io.netty:netty-tcnative-boringssl-static:2.0.56.Final",
+    "io.netty:netty-tcnative-classes:2.0.56.Final",
+    "io.netty:netty-transport-native-epoll:jar:linux-x86_64:4.1.87.Final",
+    "io.netty:netty-transport-native-unix-common:4.1.87.Final",
+    "io.netty:netty-transport:4.1.87.Final",
+    "io.opencensus:opencensus-api:0.24.0",
+    "io.opencensus:opencensus-contrib-grpc-metrics:0.24.0",
+    "io.perfmark:perfmark-api:0.25.0",
+    "junit:junit:4.12",
+    "org.apache.tomcat:annotations-api:6.0.53",
+    "org.codehaus.mojo:animal-sniffer-annotations:1.21",
+]
+
+maven.install(
+    name = "contrib_rules_jvm_deps",
+    artifacts = [
+        "com.google.code.findbugs:jsr305:3.0.2",
+        "com.google.errorprone:error_prone_annotations:2.11.0",
+        "com.google.guava:guava:30.1.1-jre",
+        "commons-cli:commons-cli:1.5.0",
+        "io.grpc:grpc-api:1.40.0",
+        "io.grpc:grpc-core:1.40.0",
+        "io.grpc:grpc-netty:1.40.0",
+        "io.grpc:grpc-services:1.40.0",
+        "io.grpc:grpc-stub:1.40.0",
+        "org.slf4j:slf4j-simple:1.7.32",
+        "com.google.googlejavaformat:google-java-format:1.15.0",
+
+        # These can be versioned independently of the versions in `repositories.bzl`
+        # so long as the version numbers are higher.
+        "org.junit.jupiter:junit-jupiter-engine:5.8.2",
+        "org.junit.jupiter:junit-jupiter-api:5.8.2",
+        "org.junit.platform:junit-platform-launcher:1.8.2",
+        "org.junit.platform:junit-platform-reporting:1.8.2",
+        "org.junit.vintage:junit-vintage-engine:5.8.2",
+
+        # Open Test Alliance for the JVM dep
+        "org.opentest4j:opentest4j:1.2.0",
+
+        # Checkstyle deps
+        "com.puppycrawl.tools:checkstyle:10.2",
+
+        # PMD deps
+        "net.sourceforge.pmd:pmd-dist:6.46.0",
+
+        # Spotbugs deps
+        "org.slf4j:slf4j-api:1.7.32",
+        "org.slf4j:slf4j-jdk14:1.7.32",
+
+        # We explicitly declare a protobuf runtime version
+        # so that it coincides with the one we use to generate the code.
```

Comment:
Not something to resolve in this PR, but I think that using the `override_targets` attribute of `maven.install` could be better than forcing the same version (but using different artifacts) here.